components/ExerciseCard.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { View, StyleSheet, Text, TextInput } from "react-native";
+import { Button, Icon } from "react-native-elements";
+import WorkoutCard from "../components/WorkoutCard";
+import { FontStyles } from "../styles/global";
+import SetDetailsView from "../screens/SetDetailsView";
+import { isTSExpressionWithTypeArguments } from "@babel/types";
```

Comment:
What is this used for?

Reply:
hmmm, weird. Not sure where it came from. Probably, I accidentally copied it. Good catch 👍